pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -44,11 +45,17 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 	platform := ic.Config.Platform.Name()
 	switch platform {
 	case aws.Name:
+		permissionGroups := []awsconfig.PermissionGroup{awsconfig.PermissionCreateBase}
```

Comment:
hmm, maybe we need to add `PermissionDeleteBase` here ??

Reply:
Ah, yes.